Makefile

```diff
@@ -950,10 +950,10 @@ lint-md-clean:
 	$(RM) -r tools/remark-preset-lint-node/node_modules
 
 lint-md-build:
-	if [ ! -d tools/remark-cli/node_modules ]; then \
-		cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
-	if [ ! -d tools/remark-preset-lint-node/node_modules ]; then \
-		cd tools/remark-preset-lint-node && ../../$(NODE) ../../$(NPM) install; fi
+	@if [ ! -d tools/remark-cli/node_modules ]; then \
+		@cd tools/remark-cli && ../../$(NODE) ../../$(NPM) install; fi
```

Comment:
I'd rather not have the @ on this line.

Also an echo to tell people why npm modules are randomly being installed would be useful (see https://github.com/nodejs/node/pull/16551)